### ISSUE TITLE
is it compatible with tokumx?

### ISSUE THREAD

**Issue by Kikobeats** [balderdashy/sails-mongo#248]
hey guys, anything of you know if [tokumx](http://www.tokutek.com/tokumx-for-mongodb/) can works fine with the adapter?

Basically tokumx is a MongoDB-like but the way that the database handle the data is different, and this reduces the number of I/O events and improvements the indexing, allow use [MVCC](https://es.wikipedia.org/wiki/Multiversion_concurrency_control)....

But is not clear for me:

1) Is it possible use a regular mongodb client? or necessary you need to use the tokumx client binary?
2) Because follow the same mongodb interface, can works fine with the adapter?  

On the other hand exist a other version called [tokudb](http://www.tokutek.com/tokudb-for-mysql/) that is the same but for mysql databases.

**Comment by zabaat** [comment]
Did you get anywhere with this?

**Comment by sailsbot** [comment]
Thanks for posting, @Kikobeats.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by devinivy** [comment]
@Kikobeats would be very interested what you learned.  As far as I know, this "should" work since the underlying driver we use is supposedly compatible with tokumx.

**Comment by Kikobeats** [comment]
because dev team didn't say nothing I didn't investigate more
